packages/block-editor/src/hooks/border.scss

```diff
@@ -3,51 +3,3 @@
 		grid-column: span 1;
 	}
 }
-
-.block-editor__border-box-control__popover,
-.block-editor__border-box-control__popover-top,
-.block-editor__border-box-control__popover-right,
-.block-editor__border-box-control__popover-bottom,
-.block-editor__border-box-control__popover-left {
-	.components-popover__content {
-		width: 282px;
-	}
-}
```

Comment:
Just noting that this will also remove the width for the border control popovers so within the block editor they are no longer consistent with the color dropdown controls.

Reply:
yeah, I was wondering whether we could just add that default width to the border-box-control entirely if it's a better default.

Reply:
I took this as a quick and dirty approach while scratching up one of the alternate border control popover PRs (https://github.com/WordPress/gutenberg/pull/40874). I agree with [Marco's comments](https://github.com/WordPress/gutenberg/pull/40874#discussion_r866699667) there it might be ok for a first pass but we should try and make this more adaptable given the goals of the components package and use outside our editors.